.github/workflows/nightly_build.yml

```diff
@@ -45,5 +45,76 @@ jobs:
       - name: "Store archive"
         uses: actions/upload-artifact@v2
         with:
-            name: ${{ env.release_name }}.tar.gz
+            name: ${{ matrix.artifact }}
             path: ${{ github.workspace }}/${{ env.release_name }}.tar.gz
+
+  publish:
+    needs: [ build ]
+    runs-on: "ubuntu-latest"
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          repository: ${{ github.repository_owner }}/glpi-project.github.io
+          token: ${{ secrets.GH_PUBLISHING_TOKEN }}
+          fetch-depth: 0
+      - name: "Make temporary folder"
+        run: |
+          rm -rf glpi/tmp
+          mkdir glpi/tmp
+      - uses: actions/download-artifact@v2
+        with:
+          name: 9.4-bugfixes
+          path: glpi/tmp
+      - uses: actions/download-artifact@v2
+        with:
+          name: 9.5-bugfixes
+          path: glpi/tmp
```

Comment:
Isn't it possible to add the `publish-nightly` step directly on the `build` job, in order to be able to use its matrix ?

Reply:
No, indeed, we need to wait for all artifacts to be built so they can be downloaded back and be published just after.
If you need to change the matrix, you'll also have to update the download-artifact steps accordingly.

Reply:
In fact, this would be possible if we could publish directly to the nightly site. But this is not the case when using GH Pages repository.